docs/pages/v2.md

```diff
@@ -445,15 +445,12 @@ Multiple APIs return the `formMethod` of a submission. In v1 they return a lower
 ```jsx
 function Something() {
   let navigation = useNavigation();
-  let fetcher = useFetcher();
```

Comment:
Fetchers always remained uppercase since we had a back-compat layer to uppercase them